app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+    <integer name="animation_base_duration">1000</integer>
```

Comment:
trochę długo trwa ta animacja, aplikacja wydaje się wolna przez to :)

Reply:
ok :)